src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/blocklist.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="umb-block-list">
+
+    <umb-load-indicator style="margin-bottom:20px;" ng-if="vm.loading"></umb-load-indicator>
+
+    <div class="umb-block-list__wrapper" ng-style="vm.listWrapperStyles">
+
+        <div ui-sortable="vm.sortableOptions" ng-model="vm.layout" ng-if="vm.loading !== true">
+
+            <div ng-repeat="layout in vm.layout track by layout.$block.key">
+
+                <button
+                    type="button"
+                    class="btn-reset umb-block-list__block--create-button"
+                    ng-click="vm.showCreateDialog($index, $event)"
+                    ng-controller="Umbraco.PropertyEditors.BlockListPropertyEditor.CreateButtonController as inlineCreateButtonCtrl"
+                    ng-mousemove="inlineCreateButtonCtrl.onMouseMove($event)"
+                >
+                    <div class="__plus" ng-style="{'left':inlineCreateButtonCtrl.plusPosX}">+</div>
+                </button>
+
+                <div class="umb-block-list__block" ng-class="{'--active':layout.$block.active}">
+
+                    <umb-block-list-scoped-block ng-if="layout.$block.config.stylesheet" class="umb-block-list__block--content blockelement__draggable-element" view="{{layout.$block.view}}" stylesheet="/{{::layout.$block.config.stylesheet}}" api="vm.blockEditorApi" block="layout.$block" index="$index">
```

Comment:
For long lines of html please please please split this up onto multiple lines, one per attribute else it's impossible to handle merge conflicts. I have done this in my wip branch already but in the future please remember this.

Reply:
yeah, this one has become a bit too long. Corrected now.